packages/indexer-common/src/indexer-management/client.ts

```diff
@@ -113,6 +115,32 @@ const SCHEMA_SDL = gql`
     location: GeoLocation
   }
 
+  type IndexingError {
+    handler: String!
```

Comment:
I believe the errors I was seeing were because this is set to non-nullable.  

Though I would also expect that if there was an `IndexingError` that it would have both of these fields. 🤔

Reply:
I've removed the `!`. This is a mistake on my part, since the schema in graph-node doesn't declare the handler as nullable: https://github.com/graphprotocol/graph-node/blob/83f65bca1ff0127693d1ef5bf624e3c4e6e7283f/server/index-node/src/schema.graphql#L69

Reply:
Thanks @Theodus!  I'll test again today.